jobs/forms.py

```diff
@@ -25,8 +19,9 @@ class MeetupForm(forms.ModelForm):
 
     class Meta:
         model = Meetup
-        exclude = ['reviewer', 'review_status', 'reviewers_comment',
-            'ready_to_publish', 'published_date', 'created', 'expiration_date'
+        fields = ['title', 'organisation', 'meetup_type', 'contact_email',
+            'website', 'city', 'country', 'description', 'is_recurring', 
+            'recurrence', 'meetup_date'
         ]
         widgets = {
             'meetup_date': forms.DateTimeInput(format='%d-%m-%Y'),
```

Comment:
I'm not entirely sure why this is not working, although I did find an old (7 yrs ago) issue about specifying formats on modelform datetimefields.  I think it can be fixed if we specify the meetup date as a custom field, and specify the default input_format and widget format.  However, this seems like a lot of extra code that shouldn't be necessary.  The old default (YYYY-MM-DD) may be better than the very american MM-DD-YYYY format, but it's up to you.